modules/@apostrophecms/ui/index.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   options: {
-    alias: 'ui'
+    alias: 'ui',
```

Comment:
This feels like the right module to put this in. The other contender was `@apostrophecms/area`